### PR TITLE
Changed mockito-all with mockito-core

### PR DESCRIPTION
**Pull request by Graeme-Miller**

### CODE REVIEW

**Comment by m4rkmckenna** [comment]
Compilation Error
```
[ERROR] /home/travis/build/cloudsoft/brooklyn-tosca/brooklyn-tosca-transformer/src/test/java/io/cloudsoft/tosca/a4c/brooklyn/Alien4CloudFacadeTest.java:[119,42] method getRequiredElementInDependencies in interface alien4cloud.component.ICSARRepositorySearchService cannot be applied to given types;
  required: java.lang.Class<T>,java.lang.String,java.util.Collection<alien4cloud.model.components.CSARDependency>
  found: java.lang.Class,java.lang.String,java.util.Set<java.lang.Object>
  reason: no instance(s) of type variable(s) T exist so that argument type java.util.Set<java.lang.Object> conforms to formal parameter type java.util.Collection<alien4cloud.model.components.CSARDependency>
```

**Comment by Graeme-Miller** [comment]
@m4rkmckenna compilation error was due to a previous issue that was fixed on master. I have rebased and it now compiles

File: brooklyn-tosca-transformer/pom.xml [hunk 1]
```diff
@@ -154,8 +154,8 @@
         </dependency>
         <dependency>
             <groupId>org.mockito</groupId>
-            <artifactId>mockito-all</artifactId>
-            <version>1.9.0</version>
+            <artifactId>mockito-core</artifactId>
+            <version>2.7.12</version>
```
Comment (m4rkmckenna) [inline brooklyn-tosca-transformer/pom.xml]: I would prefer this to be inherited from the parent pom as ie `<version>${mockito-core.version}</version>`

**Review by m4rkmckenna — APPROVED** [review]
LGTM

Minor comment feel free to ignore

**Pull request merged**